examples/controlnet/train_controlnet_flax.py

```diff
@@ -345,11 +354,16 @@ def parse_args():
         type=str,
         default=None,
         help=(
-            "A folder containing the training data. Folder contents must follow the structure described in"
-            " https://huggingface.co/docs/datasets/image_dataset#imagefolder. In particular, a `metadata.jsonl` file"
-            " must exist to provide the captions for the images. Ignored if `dataset_name` is specified."
+            "A folder containing the training dataset. By default it will use `load_dataset` method to load a custom dataset from the folder."
+            "Folder must contain a dataset script as described here https://huggingface.co/docs/datasets/dataset_script) ."
+            "If `--load_from_disk` flag is passed, it will use `load_from_disk` method instead. Ignored if `dataset_name` is specified."
         ),
     )
+    parser.add_argument(
+        "--load_from_disk",
+        action="store_true",
+        help="If True, will load a dataset that was previously saved using [`save_to_disk`] from `--train_data_dir`",
+    )
```

Comment:
Provide a link to the `load_from_disk` documentation?

Reply:
Also, the `help` seems a bit unclear to me. How does the dataset need to be saved? 

nit: I think we can just do `save_to_disk` instead of [`save_to_disk`].